resolver.go

```diff
@@ -151,22 +158,36 @@ func setCommonFlags(msg *dns.Msg) {
 	msg.RecursionAvailable = true
 }
 
+func shuffleAddr(addr []net.IP) []net.IP {
```

Comment:
Do we need to shuffle the address ?
IMHO, we can just return the multiple A records and let the Client decide the best way handle the LB
(till we have the VIP based LB). Shuffling is a poor man's DNS-RR & we can avoid that if possible.